lang/es.json

```diff
@@ -0,0 +1,47 @@
+{
+    "COMMUNITYLIGHTING": {
+        "authors": {
+            "Blitz": {
+                "lights": {
+                    "lightning": {
+                        "name": "Relámpago",
+						"shaderName": "blitzPulseTest",
```

Comment:
The `shaderName`s should not be translated. I'll add further details into a translation guide before release

Reply:
Whitespace is tabs here, where the rest of the file is spaces. Not a huge issue, I will change this before release